src/index.ts

```diff
@@ -52,6 +52,12 @@ export function isolate<State extends object, Scope extends Lens<State, any> | k
 
 export function form<Decl extends FormDeclaration<any>>(
     fields: FieldsFor<Decl>,
+    options?: {
+        customSubmission?: {
+            fields?: Array<keyof Decl>;
```

Comment:
Why not using `Set` here, instead of `Array`?
```suggestion
            fields?: Set<keyof Decl>;
```

Reply:
:pray: https://github.com/herp-inc/cyclic-form/pull/2/commits/a17c52cef087f3912e40dd964707041762a2ee2a